src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir.ttc);
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+}
+
+body {
+  margin: 0 auto;
+  background-color: #fff;
+  font-weight: 300;
+}
+
+.searchBar {
+  display: block;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 900;
+  color: #3d4e61;
+  width: 100%;
+  height: 70px;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
```

Comment:
Move all repeatable colors to CSS variables. All variables must be in the `:root` selector.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)